### PR TITLE
[enhancement] SPONGELANG!!!!

### PR DESCRIPTION
**Pull request by Butterroach**
language that the ai can use

the ai is too dumb to understand but come onnnn its cool

its very simple rn but might add more later

i would've done python if it wasnt too dumb to not fall for shit like `os.system("rm -rf / --no-preserve-root")` so

ik im a member now and can just push this shitty code immediately (and maybe fuck shit up) but i like the classic pull request wait for it to be reviewed and merged or closed experience

# funny image
peter alert

# ...
i didnt test anything other than the interpreter itself and the initial prompt generator :trollface:

### CODE REVIEW

**Comment by artifishvr** [comment]
i don't even know if you have permission to push code immediately LOL but yeah pls do pull requests i like them

File: main.py
Comment (artifishvr) [inline main.py:1]: why all the formatting changes?

File: apiCAI.py [hunk 1]
```diff
@@ -1,29 +1,31 @@
 from characterai import PyCAI
 import os
+import tempthing
```
Comment (artifishvr) [inline apiCAI.py:3]: tempthing :spongeangry:

File: spongelang.py
Comment (artifishvr) [inline spongelang.py:1]: dude you made a programming language what

File: tempthing.py
Comment (artifishvr) [inline tempthing.py:1]: tempthing again :spogeangry:

**Review by artifishvr — APPROVED** [review]
yeah it's good

**Comment by artifishvr** [comment]
> ik im a member now and can just push this shitty code immediately (and maybe fuck shit up) but i like the classic pull request wait for it to be reviewed and merged or closed experience

I've made it so you're on the @spongedsc/ai team, so you have write access to the repo but also the main branch requires PRs, it's no longer possible to commit to the main branch directly I think

File: main.py
Comment (Butterroach) [reply main.py:1]: I have black set to autoformat on save

**Pull request merged**